packages/@aws-cdk/aws-scheduler-targets-alpha/test/inspector-start-assessment-run.test.ts

```diff
@@ -0,0 +1,509 @@
+import { ScheduleExpression, Schedule } from '@aws-cdk/aws-scheduler-alpha';
+import { App, Duration, Stack } from 'aws-cdk-lib';
+import { Template } from 'aws-cdk-lib/assertions';
+import { AccountRootPrincipal, Role } from 'aws-cdk-lib/aws-iam';
+import { CfnAssessmentTarget, CfnAssessmentTemplate } from 'aws-cdk-lib/aws-inspector';
+import * as sqs from 'aws-cdk-lib/aws-sqs';
+import { InspectorStartAssessmentRun } from '../lib';
+
+describe('schedule target', () => {
+  let app: App;
+  let stack: Stack;
+  let template: CfnAssessmentTemplate;
+  const expr = ScheduleExpression.at(new Date(Date.UTC(1969, 10, 20, 0, 0, 0)));
+
+  beforeEach(() => {
+    app = new App();
+    stack = new Stack(app, 'Stack', { env: { region: 'us-east-1', account: '123456789012' } });
+    const assessmentTarget = new CfnAssessmentTarget(stack, 'MyAssessmentTarget');
+    template = new CfnAssessmentTemplate(stack, 'MyTemplate', {
+      assessmentTargetArn: assessmentTarget.attrArn,
+      durationInSeconds: 3600,
+      rulesPackageArns: ['arn:aws:inspector:us-east-1:316112463485:rulespackage/0-gEjTy7T7'],
+    });
+  });
+
+  test('creates IAM role and IAM policy for inspector assessment template in the same account', () => {
+    const inspectorTarget = new InspectorStartAssessmentRun(template, {});
```

Comment:
```suggestion
    const inspectorTarget = new InspectorStartAssessmentRun(template);
```
Can you please remove the optional parameters here as well? where possible.